### DISCUSSION TITLE
Welcome to GroupThink Discussions!

### DISCUSSION THREAD

**Discussion by Sam-Osian** [Samuel-Osian-Andrews/GroupThink#1] [Announcements]
## 👋 Hey!
Please post here if you have any suggestions or feature requests. If you have a bug report, please use the Issues tab instead. Thanks!

**Comment by jessewise99** [comment]
Hi, love what you're doing! Any update on this future plan? I would love to use it before Christmas...
"Add unify() support for demographic comparison (comparing figures between factors such as age, sex, etc.)"
